src/Greenshot.Editor/Forms/ImageEditorForm.cs

```diff
@@ -183,6 +184,16 @@ private void Initialize(ISurface surface, bool outputMade)
 
             // Workaround: As the cursor is (mostly) selected on the surface a funny artifact is visible, this fixes it.
             HideToolstripItems();
+
+            HideEmojiButtonWhenFontIsNotInstalled();
+        }
+
+        private void HideEmojiButtonWhenFontIsNotInstalled()
+        {
+            if (!FontFamily.Families.Any(f => string.Equals(f.Name, "Segoe UI Emoji", StringComparison.OrdinalIgnoreCase)))
```

Comment:
Also thinking out loud, if the FontFamily.Families are not cached, this might take a lot of resources / performance when this is requested every time the editor is opened.

Reply:
You're right! I'll add a static.